wys/api/README.md

```diff
@@ -10,90 +16,85 @@ The script can request data on any day specified by the API call.
+|`speed`|integer|Exact speed of the number of vehicles in `count`|47|
+|`count`|integer|Number of vehicles in (datetime_bin,api_id,speed) combination|2|
+|`speed_count_uid`|integer|A unique identifier for `speed_counts_agg_5kph` table. Indicates if the data has already been processed or not.|150102|
 
-`wys.speed_counts_agg_5kph` has data aggregated to 1-hour and 5 km/h bins using the `aggregate_speed_counts_one_hour_5kph()` function. Values for the speed bins are replaced by lookup table IDs.
+As described above, `wys.speed_counts_agg_5kph` has data aggregated to 1-hour and 5 km/h bins using the `aggregate_speed_counts_one_hour_5kph()` function. Values for the speed bins are replaced by lookup table IDs.
 
-|Field name|Data type|Description|Example|
+|Field name|Type|Description|Example|
 |------|------|-------|------|
-`speed_counts_agg_5kph_id`|integer|A unique identifier for the `speed_counts_agg_5kph` table|2655075
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`datetime_bin`|timestamp|Start time of the 1 houraggregated bin|2018-10-29 10:00:00
-`speed_id`|integer|A unique identifier for the 5 kph speed bin in the `speed_bins` table|5
-`count`|integer|Number of vehicles in datetime_bin/api_id/speed bin combination|7
+|`speed_counts_agg_5kph_id`|bigint|A unique identifier for the `speed_counts_agg_5kph` table|2655075|
+|`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967|
+|`datetime_bin`|timestamp|Start time of the 1 houraggregated bin|2018-10-29 10:00:00|
+|`speed_id`|integer|A unique identifier for the 5 kph speed bin in the `speed_bins` table|5|
+|`volume`|integer|Number of vehicles in datetime_bin/api_id/speed bin combination|7|
 
 ### Lookup Tables
 
-`speed_bins` is a lookup table containing all the 5km/h speed bin. Bin number 25 contains any speed over 120km/h.
+`speed_bins_old` is a lookup table containing all the 5km/h speed bin. Bin number 21 contains any speed over 100km/h.
 
-|Field name|Data type|Description|Example|
+|Field name|Type|Description|Example|
 |------|------|-------|------|
 `speed_id`|integer|A unique identifier for the `speed_bins` table|5
 `speed_bin`|integer range|Range of speeds for each speed bin. The upper limit is not inclusive.|[10-25)
 
-`locations` contains each ID used for the API, direction of traffic the sign is gathering data for, address and name of the sign. Some signs may have information missing from any one of the fields.
-
-|Field name|Data type|Description|Example|
-|------|------|-------|------|
-`api_id`|integer|ID used for the API, and unique identifier for the `locations` table|1967
-`address`|text|Address of the sign|1577 Bloor Street West
-`sign_name`|text|Name of the sign. May include address + serial number, the ward name for the Mobile WYSP, or school name for Schools WYSP|Dundas St W SB 16101191
-`dir`|text|Direction of the flow of traffic|NB
-
```

Comment:
Seems a few columns are missing...

Reply:
The lines in red were deleted in this PR